service/src/main/kotlin/fi/espoo/evaka/daycare/DaycareGroupQueries.kt

```diff
@@ -59,19 +58,19 @@ fun Database.Transaction.updateGroup(
     groupId: GroupId,
     name: String,
     startDate: LocalDate,
-    endDate: LocalDate?
+    endDate: LocalDate?,
+    jamixCustomerId: String?
 ) {
-    // language=SQL
-    val sql =
-        "UPDATE daycare_group SET name = :name, start_date = :startDate, end_date = :endDate WHERE id = :id"
-    @Suppress("DEPRECATION")
-    this.createUpdate(sql)
-        .bind("id", groupId)
-        .bind("name", name)
-        .bind("startDate", startDate)
-        .bind("endDate", endDate)
-        .execute()
-        .let { if (it != 1) throw NotFound("Group $groupId not found") }
+    createUpdate {
+            sql(
+                """
+UPDATE daycare_group
+SET name = ${bind(name)}, start_date = ${bind(startDate)}, end_date = ${bind(endDate)}, jamix_customer_id = ${bind(jamixCustomerId)}
```

Comment:
Optional nitpick: if we don't want to mix nulls and empty strings in db, we could consider using `${bind(jamixCustomerId?.takeIf { it.isNotBlank() })}` or alternatively making the column NOT NULL with old rows migrated to empty string.

Reply:
I changed the data type into nullable Integer